### PR TITLE
Fix: text block required asterisk

### PR DESCRIPTION
**Pull request by jonwaldstein**
## Description

This makes the required asterisk show up for custom text blocks

## Affects

Custom text block

## Testing Instructions

- Add a custom field
- Enable required
- Make sure asterisk shows up

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205713285814722

### CODE REVIEW

File: src/FormBuilder/resources/js/form-builder/src/blocks/fields/text/Edit.tsx [hunk 1]
```diff
@@ -3,13 +3,12 @@ import {BlockEditProps} from '@wordpress/blocks';
 
 export default function Edit({attributes}: BlockEditProps<any>) {
     const {label, isRequired, description, placeholder, defaultValue} = attributes;
-    const requiredClass = isRequired ? 'give-is-required' : '';
 
     return (
         <>
```
Comment (JoshuaHungDinh) [inline src/FormBuilder/resources/js/form-builder/src/blocks/fields/text/Edit.tsx:8]: Works for me. 
I noticed we have it wrapped in fragments. Do we need them?

**Review by JoshuaHungDinh — APPROVED** [review]
Left a short question but otherwise this is G2G.

File: src/FormBuilder/resources/js/form-builder/src/blocks/fields/text/Edit.tsx (hunk 1 above)
Comment (jonwaldstein) [reply src/FormBuilder/resources/js/form-builder/src/blocks/fields/text/Edit.tsx]: Nope! resolved b2ab1131

**Review by rickalday — APPROVED** [review]
Passed manual QA tests

**Pull request merged**